README.md

```diff
@@ -59,6 +59,7 @@ Running
 From the build directory:
 
 `ln -s ../data`
+`mkdir _ignore`
```

Comment:
an alternative would be to create an empty subdirectory `data/_ignore/` and have the relevant tests in `larch-test` write to that subdirectory. This would minimize the amount of user setup required, but involve having an empty directory.

Reply:
Added folder `data/_ignore`